_config.yml

```diff
@@ -3,8 +3,16 @@ title: ADS news, blogs, and help pages
 url: https://ui.adsabs.harvard.edu
 email: adshelp@cfa.harvard.edu
 description: All your questions will be answered here.
+
+# Social
 twitter_username: adsabs
 github_username: adsabs
```

Comment:
Can we remove these two lines, since they're below now, or are both sets needed?

Reply:
Yep agreed, just checked and they aren't being referenced in the source code, so I think safe to remove.